sys-boot/tboot/tboot-1.10.3.ebuild

```diff
@@ -17,21 +17,18 @@ IUSE="custom-cflags selinux"
 # requires patching the kernel src
 RESTRICT="test"
 
-DEPEND="app-crypt/trousers
-app-crypt/tpm-tools
-dev-libs/openssl:0=[-bindist(-)]
-"
+DEPEND="dev-libs/openssl:0=[-bindist(-)]"
 
 RDEPEND="${DEPEND}
-sys-boot/grub:2
-selinux? ( sec-policy/selinux-tboot )"
+	sys-boot/grub:2
+	selinux? ( sec-policy/selinux-tboot )"
 
-DOCS=( README COPYING CHANGELOG )
-PATCHES=( "${FILESDIR}/${PN}-1.9.11-genkernel-path.patch" )
+DOCS=( README.md COPYING CHANGELOG )
+PATCHES=( "${FILESDIR}/${PN}-1.10.3-disable-Werror.patch"
+	  "${FILESDIR}/${PN}-1.10.3-disable-strip.patch" )
 
 src_prepare() {
-	sed -i 's/ -Werror//g' Config.mk || die
-	sed -i 's/^INSTALL_STRIP = -s$//' Config.mk || die # QA Errors
+	tc-export AS LD CC CPP AR RANLIB NM OBJCOPY OBJDUMP
```

Comment:
could you look into doing this in `src_configure` instead? I assume there's no `configure` script, and it's general practice to do flag mucking in `src_configure` and not `src_prepare`.

Reply:
Its now in src_configure